ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -63,6 +62,14 @@ void RSkComponentTextInput::drawAndSubmit(){
   }
   layer()->client().notifyFlushRequired();
 }
+void RSkComponentTextInput::drawWithNoSubmit(){
+  layer()->invalidate( RnsShell::LayerPaintInvalidate);
```

Comment:
layer->invalidate  : we need to consider the current set mask while setting the PaintInvalidate mask here